src/runner.rs

```diff
@@ -224,4 +226,35 @@ When your transaction is complete, enter 'commit' or 'abort' as appropriate."#,
         }
         Ok(())
     }
+
+    pub(crate) async fn handle_status(&self) -> Result<()> {
+        let start = Instant::now();
+
+        let result = self.deps.driver.transact(|mut tx| async {
+            let result = tx
+                .execute_statement("select VALUE name from information_schema.user_tables where status='ACTIVE'")
+                .await?;
+            tx.commit(result).await
+        }).await;
```

Comment:
Maybe we should send a HTTP GET to `/ping` instead of running a transaction?

Reply:
Updated to use HTTP GET `/ping` request